src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -0,0 +1,59 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.controller.exceptions.GroupNotEmptyException;
+import com.kodilla.ecommercee.controller.exceptions.GroupNotFoundException;
+import com.kodilla.ecommercee.domain.Group;
+import com.kodilla.ecommercee.dto.GroupDto;
+import com.kodilla.ecommercee.mapper.GroupMapper;
+import com.kodilla.ecommercee.service.GroupService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * @author Dawid Nogacz on 21.08.2019

```

Comment:
Pousuwaj te komentarze - te informacje są w githubie. To jest code smell.

Reply:
Usunięte